src/main/groovy/org/boozallen/plugins/jte/util/ConfigValidator.groovy

```diff
@@ -30,13 +30,13 @@ class ConfigValidator {
         this.flowOwner = flowOwner
     }
 
-    void validate(String schemaString, LinkedHashMap config){
+    void validate(String schemaString, LinkedHashMap config, String msgPrefix = ""){
         LinkedHashMap schema = parseSchema(schemaString)
```

Comment:
thinking instead adding the extra argument .. add `getMessagePrefix(){ return msgPrefix ? "${msgPrefix} " : "" }`  and setMessagePrefix(Strng s) properties to ConfigValidator
and call `validator.messagePrefix = "Library ${libName}"` in the library injector before using validator.
Then line 64 becomes:
`new JTEException("${messagePrefix}${msg}")`

Reply:
or define another local variable that is the actual message prefix and use that.
`String actualMsgPrefix = "${msgPrefix ? "${msgPrefix} " : ""}${msg}"`